fe2-android/app/src/main/java/com/github/dedis/popstellar/repository/LAORepository.java

```diff
@@ -104,7 +105,8 @@
   // Disposable of with the lifetime of an LAORepository instance
   private final Disposable disposables;
 
-  private LAORepository(
+  @Inject
+  public LAORepository(
```

Comment:
If we create a lao, then try to click on it, it will crash the app.
It's maybe because multiple LAORepository are used instead of a singleton.

Reply:
That's right, I forgot to add the @Singleton at the top of the class.

This should be fixed now